### DISCUSSION TITLE
Use of prima migrate dev

### DISCUSSION THREAD

**Discussion by brilvio** [CodingGarden/listd#62] [Q&A] [answered]
I think we should use `prisma db push` when developing and we can run migrate when we are finished meddling with the database, or only when is ready to production, https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push#choosing-db-push-or-prisma-migrate 

Because now every time CJ or another dev wants to add a field or change the structure of database we will have a new migration file or the init file will be replaced (replacing init file throws the idea of creating a migration file in the first place through the window).

My recomendation is deleting de init file for now, only use `primsa db push` and after we are happy with the database we run `prisma migrate dev` and call init, and every PR after that changes the structure of the database , should create a new migration with a name that explain what that migration do.

What you guys think?

**Comment by w3cj** [comment, marked as answer]
Yeah this makes sense. I have been avoiding reading about this in the prisma docs, but it seems like `prisma db push` is exactly the workflow we want until we have solidified db structure.

**Comment by w3cj** [comment]
BTW. I think we should keep the init migration file for now. I still like seeing the SQL code in the codebase.
